run_hck_client.sh

```diff
@@ -371,6 +375,7 @@ ${QEMU_BIN} \
         ${TEST_BALLOON_DEVICE} \
         ${TEST_IVSHMEM_DEVICE} \
         ${TEST_RNG_DEVICE} \
+	${TEST_VIOCRYPT_DEVICE} \
```

Comment:
Looks like an issue with indentation. Please double check.

Reply:
Thanks, it should be fixed now.